packages/mobile/src/components/lineup-tile/TrackTile.tsx

```diff
@@ -122,7 +124,9 @@ export const TrackTileComponent = ({
       return
     }
     const overflowActions = [
-      !isPremium ? OverflowAction.ADD_TO_PLAYLIST : null,
+      !isPremiumContentEnabled || !isPremium
```

Comment:
ha, fun one... theoretically tracks aren't visible if premium content isn't enabled, so i don't think we need this check here?